### PR TITLE
Enable UFMT on test_indexing&test_view_ops

### PR DESCRIPTION
**Pull request by MatrixPlayer**
Part of https://github.com/pytorch/pytorch/issues/123062

cc @ezyang

### CODE REVIEW

**Comment by ezyang** [comment]
This diff doesn't seem right

```
diff --git b/test/test_indexing.py a/test/test_indexing.py
index f34fa4c5669..21c4624e23b 100644
--- b/test/test_indexing.py
+++ a/test/test_indexing.py
@@ -861,14 +861,12 @@ class TestIndexing(TestCase):
         )
         uint8Indices = torch.tensor([1, 0, 0], dtype=torch.uint8, device=device)
         with warnings.catch_warnings(record=True) as w:
-            v1 = v[boolIndices]
-            v2 = v[uint8Indices]
-            self.assertEqual(v1.shape, v2.shape)
-            self.assertEqual(v1, v2)
+            self.assertEqual(v[boolIndices].shape, v[uint8Indices].shape)
+            self.assertEqual(v[boolIndices], v[uint8Indices])
             self.assertEqual(
                 v[boolIndices], tensor([True], dtype=torch.bool, device=device)
             )
-            self.assertEqual(len(w), 1)
+            self.assertEqual(len(w), 2)
 
     def test_bool_indices_accumulate(self, device):
         mask = torch.zeros(size=(10,), dtype=torch.bool, device=device)
@@ -887,10 +885,9 @@ class TestIndexing(TestCase):
         v = torch.randn(5, 7, 3, device=device)
         mask = torch.ByteTensor([1, 0, 1, 1, 0]).to(device)
         with warnings.catch_warnings(record=True) as w:
-            res = v[mask]
-            self.assertEqual(res.shape, (3, 7, 3))
-            self.assertEqual(res, torch.stack([v[0], v[2], v[3]]))
-            self.assertEqual(len(w), 1)
+            self.assertEqual(v[mask].shape, (3, 7, 3))
+            self.assertEqual(v[mask], torch.stack([v[0], v[2], v[3]]))
+            self.assertEqual(len(w), 2)
 
         v = torch.tensor([1.0], device=device)
         self.assertEqual(v[v == 0], torch.tensor([], device=device))
```

**Comment by MatrixPlayer** [comment]
> This diff doesn't seem right
> 
> ```
> diff --git b/test/test_indexing.py a/test/test_indexing.py
> index f34fa4c5669..21c4624e23b 100644
> --- b/test/test_indexing.py
> +++ a/test/test_indexing.py
> @@ -861,14 +861,12 @@ class TestIndexing(TestCase):
>          )
>          uint8Indices = torch.tensor([1, 0, 0], dtype=torch.uint8, device=device)
>          with warnings.catch_warnings(record=True) as w:
> -            v1 = v[boolIndices]
> -            v2 = v[uint8Indices]
> -            self.assertEqual(v1.shape, v2.shape)
> -            self.assertEqual(v1, v2)
> +            self.assertEqual(v[boolIndices].shape, v[uint8Indices].shape)
> +            self.assertEqual(v[boolIndices], v[uint8Indices])
>              self.assertEqual(
>                  v[boolIndices], tensor([True], dtype=torch.bool, device=device)
>              )
> -            self.assertEqual(len(w), 1)
> +            self.assertEqual(len(w), 2)
>  
>      def test_bool_indices_accumulate(self, device):
>          mask = torch.zeros(size=(10,), dtype=torch.bool, device=device)
> @@ -887,10 +885,9 @@ class TestIndexing(TestCase):
>          v = torch.randn(5, 7, 3, device=device)
>          mask = torch.ByteTensor([1, 0, 1, 1, 0]).to(device)
>          with warnings.catch_warnings(record=True) as w:
> -            res = v[mask]
> -            self.assertEqual(res.shape, (3, 7, 3))
> -            self.assertEqual(res, torch.stack([v[0], v[2], v[3]]))
> -            self.assertEqual(len(w), 1)
> +            self.assertEqual(v[mask].shape, (3, 7, 3))
> +            self.assertEqual(v[mask], torch.stack([v[0], v[2], v[3]]))
> +            self.assertEqual(len(w), 2)
>  
>          v = torch.tensor([1.0], device=device)
>          self.assertEqual(v[v == 0], torch.tensor([], device=device))
> ```

fix this issue, please retrigger CI, thanks :-) @ezyang

**Comment by ezyang** [comment]
You are still inlining the masking calls, whereas previously the test ran the mask call once and then tested the result

```
diff --git b/test/test_indexing.py a/test/test_indexing.py
index f34fa4c5669..9b09ad06a8b 100644
--- b/test/test_indexing.py
+++ a/test/test_indexing.py
@@ -861,10 +861,8 @@ class TestIndexing(TestCase):
         )
         uint8Indices = torch.tensor([1, 0, 0], dtype=torch.uint8, device=device)
         with warnings.catch_warnings(record=True) as w:
-            v1 = v[boolIndices]
-            v2 = v[uint8Indices]
-            self.assertEqual(v1.shape, v2.shape)
-            self.assertEqual(v1, v2)
+            self.assertEqual(v[boolIndices].shape, v[uint8Indices].shape)
+            self.assertEqual(v[boolIndices], v[uint8Indices])
             self.assertEqual(
                 v[boolIndices], tensor([True], dtype=torch.bool, device=device)
             )
@@ -887,9 +885,8 @@ class TestIndexing(TestCase):
         v = torch.randn(5, 7, 3, device=device)
         mask = torch.ByteTensor([1, 0, 1, 1, 0]).to(device)
         with warnings.catch_warnings(record=True) as w:
-            res = v[mask]
-            self.assertEqual(res.shape, (3, 7, 3))
-            self.assertEqual(res, torch.stack([v[0], v[2], v[3]]))
+            self.assertEqual(v[mask].shape, (3, 7, 3))
+            self.assertEqual(v[mask], torch.stack([v[0], v[2], v[3]]))
             self.assertEqual(len(w), 1)
 
         v = torch.tensor([1.0], device=device)
```

I'm sure it also works to test it this way, but it's the principle of the thing (lints should not change code meaning)

**Comment by MatrixPlayer** [comment]
I'm not sure why this code snippet is inlined, anyway, revert these changes

**Comment by ezyang** [comment]
clean

**Review by ezyang — APPROVED** [review]
